src/resources/views/components/card.blade.php

```diff
@@ -3,7 +3,18 @@
 @endphp
 
 <div x-data="tallstackui_card(@js($initializeMinimized))" @class($personalize['wrapper.first']) x-show="show">
-    <div @class($personalize['wrapper.second'])>
+    <div @class([
+            $personalize['wrapper.second'], 
+            $personalize['wrapper.overlay'] => $overlay
+        ]) @if($overlay) style="background-image: url('{{ $image }}'); background-size: cover;" @endif >
```

Comment:
We don't define `style` in our components without a good reason. I'll be reviewing all of our components to remove any unnecessary styles soon, for V2. Can you provide us with a better approach for this or can we leave it up to the dev:

```blade
<x-card ... style="...">
```